Create basic game structure

DavidB09 opened the pull request:
New Logic:
- Prompts user for game mode and checks for correct input
- Gets user input for move and checks for correct input
- Displays new board with move made

DavidB09 committed:
Create basic game structure

DavidB09 committed:
Add end game display, generate cpu move logic

averms reviewed (changes requested):
Also discrepancies with the style guidelines:

- Indentation and alignment should be done with tabs
- Instructions shouldn't be indented more than once, labels at all

Other than that, it looks great

DavidB09 committed:
Load byte using declared data, change use of register  to

DavidB09 committed:
Merge branch 'trunk' into gameplay

DavidB09 committed:
Merged new code with old code for gameplay

DavidB09 reviewed (commented):
I attempted to merge the code together. The printing function and user input seems to work fine; however, the logic for game status seems to not work correctly. Perhaps the changes altered some of the necessary logic?

DavidB09 committed:
Redo get_game_status and add logic for asking user which character to print

DavidB09 committed:
Change la to lb, add comments

qasimb03 committed:
Updated tic_tac_toe.asm

Added simple prompts that tell the user which player's turn it is (whether playing against another player or CPU). Also formatted prompts that ask user to enter 1 or 2 for playing against CPU or player and for choosing a character so it is easier for the user to read.

DavidB09 committed:
Change order of display

DavidB09 committed:
Add comments

DavidB09 committed:
Fix whitespace
